src/main/webapp/dev/js/common/questionRank.es6

```diff
@@ -220,10 +220,18 @@ function bindRankEvents() {
         });
 }
 
+function removeInvalidFeedbackPathsForRankRecipientQuestions() {
```

Comment:
Maybe I miss something, but how does this code know to target only "rank recipients" type of question and not including "rank options"?

Reply:
Sorry about that 😅 Will fix it!